lib/monocle.js

```diff
@@ -180,8 +187,16 @@ var buildFullPath = function(base, path) {
 };
 
 function buildQuery(options) {
-    var query = {};
+    var query = {}
+
+    if (options && options.query && typeof options.query === "object") {
+        for (var i in options.query) {
```

Comment:
Use [`querystring.stringify()`](https://nodejs.org/api/querystring.html#querystring_querystring_stringify_obj_sep_eq_options) to ensure keys/values are properly encoded.